### PR TITLE
Updated description & example usage

### PR DESCRIPTION
**Pull request by margaretmz**
Made changes per issue #25844:
- Updated description to mention tf.Keras model. 
- Added the missing line in the example usage for SavedModel and tf.keras model.

### CODE REVIEW

File: tensorflow/lite/python/lite.py [hunk 1]
```diff
@@ -291,7 +291,7 @@ def convert(self):
 class TFLiteConverter(object):
   """Convert a TensorFlow model into `output_format` using TOCO.
 
-  This is used to convert from a TensorFlow GraphDef or SavedModel into either a
+  This is used to convert from a TensorFlow GraphDef, SavedModel or tf.keras model into either a
```
Comment (gargn) [inline tensorflow/lite/python/lite.py]: nit: Can you fix the formatting (i.e. new line is within 80 characters).

**Review by gargn — CHANGES_REQUESTED** [review]

File: tensorflow/lite/python/lite.py (hunk 1 above)
Comment (margaretmz) [reply tensorflow/lite/python/lite.py]: Updated new line to be within 80 characters. Thanks for your review.

**Review by gargn — APPROVED** [review]

**Comment by margaretmz** [comment]
This PR has been approved but 2 checks failed. What's the next step on this?

**Comment by rthadur** [comment]
@margaretmz could you please check build errors

**Comment by margaretmz** [comment]
@dynamicwebpaige please advise the next steps on this PR, which has been approved a long time ago but 2 checks were not successful. Thanks!

**Pull request merged**